iCubGenova09/wrappers/inertials/head-imuFilter.xml

```diff
@@ -3,7 +3,7 @@
 
 
     <device xmlns:xi="http://www.w3.org/2001/XInclude" name="head-imuFilter" type="imuFilter">
-        <param name="period">       20                  </param>
+        <param name="period">       2                  </param>
```

Comment:
The hardware updates values of the inertials only every 10 ms, so the best period of the wrapper is 10 ms. To have 2 ms is not wrong, but it may stress the system more than we need.

This comment applies to all the wrappers inside folder `inertials`.